frontend/src/routes/Governance/common/util.tsx

```diff
@@ -492,19 +494,61 @@ export function getPolicyRemediation(policy: Policy | undefined) {
   }
 
   remediationAggregation = remediationAggregation.trim().replaceAll(' ', '/')
-  if (remediationAggregation === '') {
-    return '-'
+
+  if (rootRA === REMEDIATION_ACTION.ENFORCE) {
+    return remediationAggregation.includes(REMEDIATION_ACTION.INFORM_ONLY) ? rootRA.concat('/informOnly') : rootRA
   }
-  if (policy.spec?.remediationAction) {
-    if (remediationAggregation.includes('informOnly')) {
-      if (remediationSet.size > 1) {
-        return policy.spec.remediationAction.concat('/informOnly')
-      }
-      return 'informOnly'
+
+  if (rootRA) {
+    if (remediationAggregation.includes(REMEDIATION_ACTION.INFORM_ONLY)) {
+      rootRA = remediationSet.size > 1 ? rootRA.concat('/informOnly') : REMEDIATION_ACTION.INFORM_ONLY
+    }
+  } else {
+    rootRA = remediationAggregation
+    // empty root remediation and template remedation all empty
+    if (!rootRA) {
+      return '-'
+    }
+  }
+
+  let remediationOverride = ''
+  let allOverridden = propagatedPolicies.length !== 0
+  propagatedPolicies.forEach((propaPolicy) => {
+    const propaPolicyRA = propaPolicy.spec.remediationAction
+    if (!propaPolicyRA) {
+      allOverridden = false
+      return
     }
-    return policy.spec.remediationAction
+    if (propaPolicyRA === rootRA.replace('/informOnly', '')) {
+      allOverridden = false
+      return
+    }
+
+    if (!remediationOverride) {
+      remediationOverride = REMEDIATION_ACTION.ENFORCE
+    }
+  })
+
+  if (allOverridden) {
+    return rootRA.includes('informOnly')
+      ? REMEDIATION_ACTION.INFORMONLY_ENFORCE_OVERRIDDEN
+      : REMEDIATION_ACTION.ENFORCE_OVERRIDDEN
```

Comment:
What about the case where you just have a single policy template with `informOnly`? Then the override wouldn't have an effect.

Reply:
but user added enforce on root?

Reply:
Right. If I have a policy with no remediation action, a single policy template with `informOnly`, and overrides set on all policies, then the enforce has no effect. It's still just `informOnly`. So maybe earlier in this function, you can just return `informOnly` and not check for overrides if all the policy templates are `informOnly`.

Reply:
how about this line 503
```
  if (rootRA) {
    if (remediationAggregation.includes(REMEDIATION_ACTION.INFORM_ONLY)) {
      if (remediationSet.size > 1 ) {
 rootRA= rootRA.concat('/informOnly') 
       }else{
            return  REMEDIATION_ACTION.INFORM_ONLY
        }
    }
```

If only one informOnly or all informOnly, override doesn't affect at all so